### PR TITLE
feat: improve profile management

### PR DESCRIPTION
**Pull request by alex16384**
Added `COMPOSE_PROFILES` environment variable to make it easier to manage containers via docker compose cli

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -10,6 +10,8 @@ PORT_L2_EXECUTION_ENGINE_P2P=30306
 PORT_PROVER_SERVER=9876
 PORT_PROMETHEUS=9091
 PORT_GRAFANA=3001
+# Comma-separated profiles to be used by default (l2_execution_engine,proposer,prover)
+COMPOSE_PROFILES=l2_execution_engine,proposer
```
Comment (RogerLamTd) [inline .env.sample:14]: Maybe default to just l2_execution_engine?

**Review by RogerLamTd — CHANGES_REQUESTED** [review]

**Comment by alex16384** [comment]
good idea, thanks

File: .env.sample (hunk 1 above)
Comment (alex16384) [reply .env.sample]: good idea, thanks

**Comment by RogerLamTd** [comment]
@alex16384 please change base branch to hekla

**Comment by alex16384** [comment]
> @alex16384 please change base branch to hekla

done

**Review by RogerLamTd — APPROVED** [review]

**Pull request merged**